modules/social_features/social_activity/social_activity.tokens.inc

```diff
@@ -14,14 +14,19 @@ use Drupal\node\NodeInterface;
  * Implements hook_token_info().
  */
 function social_activity_token_info() {
-  $additional_information['additional_information'] = [
+  $tokens['additional_information'] = [
```

Comment:
//cc @navneet0693 have a look for our email notification PR, we need to make sure this token works when merging this back in our PR later.